frontend/packages/console-shared/src/components/dashboard/launcher-card/launcher-card.scss

```diff
@@ -0,0 +1,12 @@
+.co-launcher-card__body {
+  display: flex;
+  flex-direction: column;
+}
+
+.co-launcher-card__item {
```

Comment:
can you use the spacer variable here? `var(--pf-global--spacer--sm)`

Reply:
sure, using variable now